### PR TITLE
Service Map - added intros for heading sections

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-map.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-19/docs-ref-services/service-map.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180133308407-119%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/service-map.md [hunk 1]
```diff
@@ -16,17 +16,25 @@ ms.service: Service Map
...
 
 Learn more about [Azure Service Map](https://docs.microsoft.com/en-us/azure/operations-management-suite/operations-management-suite-service-map).
 
-## Management Package
-### Install the npm module
+## Management package
+
+### Install npm module
+
+Install the Azure Service map npm module
```
Comment (clarkio) [inline docs-ref-services/service-map.md]: Should this be `Azure Service Map npm package`? Capitalize the full product name and replace "module" with "package"?

File: docs-ref-services/service-map.md [hunk 2]
```diff
@@ -16,17 +16,25 @@ ms.service: Service Map
 # Azure Service Map packages for Node.js
```
Comment (clarkio) [inline docs-ref-services/service-map.md]: Since it's only one package should this read `Azure Service Map package for Node.js` for now?

Comment (clarkio) [inline docs-ref-services/service-map.md]: Also, let's capitalize `Package` so that is consistent across the docs as this is the title of the document.

**Comment by johnpapa** [comment]
please merge @TomArcher 

#sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-map.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-19/docs-ref-services/service-map.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180339308895-119%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-map.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-19/docs-ref-services/service-map.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180356510031-119%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-map.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-19/docs-ref-services/service-map.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180400291447-119%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-map.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-19/docs-ref-services/service-map.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180412574775-119%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@johnpapa @clarkio merged

**Pull request merged**